modules/dynamic-plugins/proc-load-plugin-oci-image.adoc

```diff
@@ -20,8 +20,27 @@ plugins:
     package: oci://quay.io/example/image:v0.0.1!backstage-plugin-myplugin
 ----
 --
-
-. Configure authentication for private registries by setting the `REGISTRY_AUTH_FILE` environment variable to the path of the registry configuration file. For example, `~/.config/containers/auth.json` or `~/.docker/config.json`.
+. Configure authentication for private registries by logging in to the container image registry.
```

Comment:
I think this should actually be the first step (if the plugin needs to be pulled from an authenticated registry). Otherwise, for Operator-backed instances, if the user first updates their `dynamic-plugins.yaml` ConfigMap and saves it, the Operator might restart the RHDH application automatically, which will likely cause the dynamic plugin installation to fail until authentication is properly configured.

Reply:
Also, maybe all these steps could be underneath a single "Optional. If the plugin needs to be pulled from an authenticated registry, complete the following steps:" bullet point ?